packages/core/src/lib/quote/quote.spec.ts

```
@@ -7,13 +7,13 @@ import { Quote } from './quote';
 import { describe, expect, it } from 'vitest';
 
 describe('quote', () => {
-  const quoteUrl = 'https://price.garden.finance';
+  const quoteUrl = 'https://pricev2.garden.finance';
   const quote = new Quote(quoteUrl);
 
-  it('should get quote for evm-strk', async () => {
+  it.only('should get quote for evm-strk', async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)